Modix.Data/Models/Core/DesignatedChannelType.cs

```diff
@@ -28,6 +28,10 @@ public enum DesignatedChannelType
         /// <summary>
         /// Defines a channel to which starred messages are sent.
         /// </summary>
-        Starboard
+        Starboard,
+        /// <summary>
+        /// Defines a channel that should be included when calculating user participation.
+        /// </summary>
+        CountsTowardsParticipation
```

Comment:
So, this is pretty annoying, but there's also a TypeScript enum that you need to modify in order for the designation to show up in the website: https://github.com/discord-csharp/MODiX/blob/master/Modix/ClientApp/src/models/moderation/ChannelDesignation.ts

Reply:
Updated in 25999e4.